cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -98,6 +98,54 @@ def counts_to_results(
     return result
 
 
+def _validate_cirq_circuit(circuits: Any) -> None:
```

Comment:
nits: plural name, and avoid using `Any` (which is made unnecessary by the `isinstance` checks)
```suggestion
def _validate_cirq_circuits(circuits: object) -> None:
```